src/pages/docs/matchmaker/guides/custom-games.mdx

```diff
@@ -6,25 +6,37 @@ the configurations available with custom lobbies.
 
 ## Creation
 
-To create custom lobbies, you must first enable them in the version config:
+To create custom lobbies, you must first enable them in your version config:
 
 <CodeGroup title='rivet.yaml'>
-  ```yaml matchmaker: game_modes: default: actions: create: enabled: true # Public custom lobbies will be
-  listed in the response to `lobbies.list`, private custom # lobbies will not enable_public: true
-  enable_private: true max_lobbies_per_identity: 5 # Optional ```
+  ```yaml
+  # ...
+
+  matchmaker:
+    game_modes:
+      default:
+        actions:
+          create:
+            enabled: true
+            # Public custom lobbies will be listed in the response to `lobbies.list`, private custom
+            # lobbies will not
+            enable_public: true
+            enable_private: true
+            max_lobbies_per_identity: 5 # Optional
```

Comment:
does yaml like this? i try to loosely follow strictyaml which would not like this. can we put "optional" on the line above?

Reply:
what part about this is wrong? yaml checkers say its all valid

Reply:
looking it up, i think it is valid.

my problem is there's no clear delimiter of what the value actually is. are there spaces after "true" before the "#"? this is really a tiny nit, but just like to avoid ambiguity.

Reply:
The comment would be a different color. Did you want it to be on a different line?

Reply:
meh whatever it's nbd